stable/cert-exporter/README.md

```diff
@@ -30,7 +30,7 @@ The following table lists the configurable parameters of the cert-exporter chart
 
 | Parameter                            | Description                                                                       | Default                                    |
 | ------------------------------------ | --------------------------------------------------------------------------------- | ------------------------------------------ |
-| `image.repository`                   | The image repository to pull from                                                 | `registry.suse.com/caasp/v4/cert-exporter` |
+| `image.repository`                   | The image repository to pull from                                                 | `registry.suse.com/caasp/v4.5/cert-exporter` |
```

Comment:
nit: could you help to align this?
![Screenshot-2020-08-27-15:40:45](https://user-images.githubusercontent.com/22583541/91412050-cd3a6880-e87b-11ea-939a-25166456563a.png)